### PR TITLE
Completed hibernate06

### PR DESCRIPTION
**Pull request by Vedroid**

### CODE REVIEW

File: src/main/java/ua/vedroid/cinema/model/Ticket.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "movie_session_id")
+    private MovieSession movieSession;
+    @OneToOne
+    @JoinColumn(name = "user_id")
+    private User user;
```
Comment (Mandybur800) [inline src/main/java/ua/vedroid/cinema/model/Ticket.java]: Are you sure about `@OneToOne` for `User`?

**Review by Mandybur800 — APPROVED** [review]

File: src/main/java/ua/vedroid/cinema/service/ShoppingCartService.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import ua.vedroid.cinema.model.ShoppingCart;
+import ua.vedroid.cinema.model.User;
+
+public interface ShoppingCartService extends GenericService<ShoppingCart> {
+    /**
+     * This method is responsible for adding a Ticket to the ShoppingCart
+     *
+     * @param movieSession contains the information required for the ticket
+     * @param user         - the User who wants to buy the ticket for a specific movieSession
+     */
```
Comment (KseniiaMakarova) [inline src/main/java/ua/vedroid/cinema/service/ShoppingCartService.java]: let's remove this

File: src/main/java/ua/vedroid/cinema/service/impl/ShoppingCartServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    public List<ShoppingCart> getAll() {
+        return shoppingCartDao.getAll();
+    }
+
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        Ticket ticket = new Ticket();
+        ticket.setMovieSession(movieSession);
+        ticket.setUser(user);
+        ticketDao.add(ticket);
```
Comment (KseniiaMakarova) [inline src/main/java/ua/vedroid/cinema/service/impl/ShoppingCartServiceImpl.java]: if there is no shopping cart for the user - we still save ticket to the db, is it ok?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]
see one more comment above

**Pull request merged**